src/utilities/activity-utils.ts

```diff
@@ -75,9 +65,21 @@ export const getPageSideTipEmbeddables = (activity: Activity, currentPage: Page)
     }
     return sidetips || [];
   }
-  // else {
-  //   return currentPage.sections.embeddables.filter((e: any) => isEmbeddableSideTip(e));
-  // }
+  else {
+    const sidetips: EmbeddableType[] = [];
+    const sectionsInPage = currentPage?.sections;
+    if (currentPage) {
+      for (let section = 0; section < sectionsInPage.length; section ++) {
+        for (let embeddableNum = 0; embeddableNum < sectionsInPage[section].embeddables.length; embeddableNum++) {
+          const embeddableWrapper = sectionsInPage[section].embeddables[embeddableNum];
+          if (isEmbeddableSideTip(embeddableWrapper)) {
+            sidetips.push(embeddableWrapper);
+          }
+        }
+      }
```

Comment:
Yeah, I agree with @emcelroy. Both loops do the same thing. The for loops you are using are more explicit, and arguably more efficient because they don't involve creating anonymous functions. 

But the foreach style is shorter, and much easier to read.